tools/telemetry-generator/src/handlers/appInsightsExecutionTimeTestHandler.ts

```diff
@@ -33,17 +38,25 @@ module.exports = function handler(fileData, telemetryClient: TelemetryClient): v
 				},
 			});
 		} catch (error) {
-			console.error(`failed to emit metric ${arithmeticMeanMetricName}`, error);
+			console.error(`failed to emit metric '${arithmeticMeanMetricName}'`, error);
 		}
 
 		const marginOfErrorMetricName = `${fileData.suiteName}_${testData.benchmarkName}_marginOfError`;
 		try {
+			let value = testData.customData["Margin of Error"];
```

Comment:
sample value is like: ±0.99%

Reply:
Looks like it's formatted this way on purpose. Also it's in this format before customBenchmark.
https://github.com/microsoft/FluidFramework/pull/21730

Reply:
So, I just ran the perf benchmarks for the sequence package and "Margin of Error" has no formatting:

![image](https://github.com/user-attachments/assets/83607ce3-3cd6-457b-b82b-53f79bc57b8f)

Which input file did you use that seems to have them?